### PR TITLE
MicroMessenger bot

### PR DESCRIPTION
**Pull request by MikeVL**
UserAgent
```
Mozilla/5.0 (Linux; Android 7.0; FRD-AL00 Build/HUAWEIFRD-AL00; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/53.0.2785.49 Mobile MQQBrowser/6.2 TBS/043602 Safari/537.36 MicroMessenger/6.5.16.1120 NetType/WIFI Language/zh_CN
```

### CODE REVIEW

File: src/Fixtures/Crawlers.php [hunk 1]
```diff
@@ -37,6 +37,7 @@ class Crawlers extends AbstractProvider
         '^Jetty\/',
         '^Mail\/',
         '^Mget',
+        '^MicroMessenger\/',
```
Comment (JayBizzle) [inline src/Fixtures/Crawlers.php]: Hi,

Thanks for the PR. You need to remove the `^` from this line to get the tests to pass.

Thanks 👍

**Comment by JayBizzle** [comment]
Hi @MikeVL 

If you can resolve my comment above, i'll merge this ASAP.

👍

**Comment by dmyers** [comment]
Are you sure this is an actual bot? I have registered users with application sessions attached to clicks with this user agent which it seems like this could be some kind of browser or something because of the session data.

**Comment by JayBizzle** [comment]
@dmyers looks like you may be correct. Would you like to create a PR to rectify this?

Thanks 👍

**Comment by tractorcow** [comment]
I wonder if exclusions, crawlers could be injectable so we could adjust them on a case by case basis? I would have to subclass CrawlerDetect if I were to add a manual exclusion.

**Comment by tractorcow** [comment]
I actually don't think excluding MicroMessenger from the crawler list would be wise, despite it hitting regular traffic. A lot of badly behaved bots do mimic that user agent, so I would probably agree with the status quo for the default crawler list.

However, in cases where wechat browsers should be selectively supported, it makes sense to allow a per-application exclusion list.

I'll PR a feature shortly.

**Comment by tractorcow** [comment]
PR at https://github.com/JayBizzle/Crawler-Detect/pull/384

**Pull request merged**